modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,138 @@
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, custom image parameters in cluster tasks are deprecated. You must run an operator-provided cluster task with a custom image. The following parameters must be configured in the custom image:
+
++
+[source,yaml]
+----
+buildah task doesn't expose "BUILDER_IMAGE" param
+git-clone task doesn't expose "gitInitImage" param
+kn task doesn't expose "kn-image" param
+kn-apply task doesn't expose "KN_IMAGE" param
+maven task doesn't expose "MAVEN_IMAGE" param
+s2i-* tasks don't expose "BUILDER_IMAGE" param
+tkn task doesn't expose "TKN_IMAGE" param
+----
+
+[id="known-issues-1-9_{context}"]
+== Known issues
+
+// .Operator
+
+* The `chains-secret` and `chains-config` config map are removed after you uninstall the Red Hat OpenShift Pipelines operator. As they contain user data, they should be preserved and not deleted.
+// https://issues.redhat.com/browse/SRVKP-2396
+
+// .PAC
+
+* When running the `tkn pac` set of commands on Windows, you may receive the following error message: Command finished with error: not supported by Windows.
+
+Workaround: Set the `NO_COLOR` environment variable to `true`.
+// https://issues.redhat.com/browse/SRVKP-2197
+
+* Running the `tkn pac resolve -f <filename> | oc create -f` command may not provide expected results, if the `tkn pac resolve` command uses a templated parameter value to function.
+
+Workaround: To mitigate this issue, save the output of `tkn pac resolve` in a temporary file by running the `tkn pac resolve -f <filename> > tempfile.yaml` command and then run the `oc create -f tempfile.yaml` command. For example, `tkn pac resolve -f <filename> -o /tmp/pull-request-resolved.yaml && kubectl create -f /tmp/pull-request-resolved.yaml`.
+
+[id="fixed-issues-1-9_{context}"]
+== Fixed issues
+
+// .Pipelines
+
+* Before this update, after replacing an empty array, the original array returns an empty string rendering the paramaters inside the invalid. With this update, this issue is resolved and the original array return as empty. 
+// Vincent Demeester
+
+* Before this update, if duplicate secrets are present in a service account for a pipelines run, it resulted in failure in task pod creation. With this update, this issue is resolved and the task pod is created successfully even if duplicate secrets were present in a service account.
+// Vincent Demeester
+
+* Before this update, by looking at the TaskRun's `spec.StatusMessage` field, users could not distinguish whether the `TaskRun` had been cancelled by the user or by a `PipelineRun` that was part of it. With this update, this issue is resolved and users can distinguish the status of the `TaskRun` by looking at the TaskRun's `spec.StatusMessage` field.
+// Vincent Demeester
+
+* Before this update, you could not set some fields of `timeouts`. With this update this issue is resolved, and you can specify some fields of `timeouts` and let the rest of the fields be default.
```

Comment:
This sentence is very hard to understand. Is it certain fields that couldn't be set, or was it an "all-or-nothing" case? this needs clarification

Reply:
As best as I can understand, it means that only some fields can be set and the rest are default.

Reply:
@rupalibehera - Could you please take a look at this comment if it is technically accurate?

Reply:
@vdemeester can you please take a look here